addons/sale_loyalty/models/sale_order.py

```diff
@@ -153,6 +164,18 @@ def action_confirm(self):
         for coupon, change in self.filtered(lambda s: s.state != 'sale')._get_point_changes().items():
             coupon.points += change
         res = super().action_confirm()
+        # Prioritize any action from super()
+        if res and isinstance(res, bool) and has_claimable_rewards:
```

Comment:
1) Should we check claimable rewards before super or is it also okay to check after?
2) Do we need to check res as it is True or not empty dict?
```suggestion
        if len(self) == 1 and isinstance(res, bool) and bool(self._get_claimable_rewards()):
```